static/beta/stage/modules/fed-modules.json

```diff
@@ -18,21 +18,21 @@
     },
     "userPreferences": {
         "manifestLocation": "/apps/user-preferences/fed-mods.json",
-        "defaultDocumentTitle": "User Preferences | User Preferences",
+        "defaultDocumentTitle": "My Notification Preferences | User Preferences",
```

Comment:
Actually, looking at https://issues.redhat.com/browse/RHCLOUD-29464 we probably want this to be `Notification Preferences`

Reply:
Yup that is correct. The Title is `Notifications Preferences` but the url is `my-notification-preferences`